api/video_codecs/video_encoder_factory.h

```diff
@@ -89,7 +89,7 @@ class VideoEncoderFactory {
     // specified format is supported. Returns false if scalability_mode is
     // specified.
     CodecSupport codec_support;
-    if (!scalability_mode) {
```

Comment:
wouldn't this return not supported when scalability mode is unset? should the if check be removed instead?

Reply:
makes sense